assets/package.json

```diff
@@ -11,14 +11,15 @@
     "watch": "webpack --mode development --watch",
     "build": "npm run clean && npm run lint && npm run test && npm run deploy",
     "clean": "del --force coverage/** ../priv/static",
-    "lint": "npx eslint src test",
+    "lint": "eslint src test",
```

Comment:
prettier & eslint are installed locally, not sure why we were running via npx. This should make it quicker